src/pages/markets/static/content/_cfds.tsx

```diff
@@ -119,6 +121,12 @@ export const synthetic_cfds: TMarketContent = {
         mobile_col: 2,
     },
     content: [
+        {
+            id: 'drift-switch-indices',
+            title: '_t_Drift switch indices_t_',
```

Comment:
Minor issue: In figma these titles have Ubuntu font family, so might have to change these and the rest

Reply:
@konstantinosG-deriv, should i change the font family?

Reply:
done